### PR TITLE
hotfix: GDCPE-1527: DataDog Service Catalog improvements

### PR DESCRIPTION
**Pull request by a-collet**
## Description
- Updating @datadog/browser-rum
- Correcting datadogRum service name

## Checklist

- [ ] Added proper unit tests
- [ ] Left proper TODO messages for any remaining tasks
- [ ] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

### CODE REVIEW

**Review by bilalbaqar — APPROVED** [review]
LGTM

File: packages/portal-proto/src/pages/_app.tsx [hunk 1]
```diff
@@ -40,14 +40,13 @@ datadogRum.init({
...
   site: "datadoghq.com",
-  service: "portal-2.0",
+  service: "portal2",
 
   // Specify a version number to identify the deployed version of your application in Datadog
   // version: '1.0.0',
   sessionSampleRate: 100,
   sessionReplaySampleRate: 0,
   trackUserInteractions: true,
-  trackFrustrations: true,
```
Comment (bilalbaqar) [inline packages/portal-proto/src/pages/_app.tsx:52]: Why remove this?

Comment (a-collet) [reply packages/portal-proto/src/pages/_app.tsx:52]: With the upgrade from v4 to v5 [Frustration signals are collected automatically](https://docs.datadoghq.com/real_user_monitoring/guide/browser-sdk-upgrade/#frustration-signals-are-collected-automatically).

Comment (craigrbarnes) [reply packages/portal-proto/src/pages/_app.tsx:52]: Is this meant to be added to `master`? It's not the current code branch `develop` is, or do you mean to update the production code? which is `release/2.14.3`?

**Pull request closed without merging**